<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestMainWindow</class>
 <widget class="QMainWindow" name="TestMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="_treeWidget">
      <column>
       <property name="text">
        <string>1</string>
       </property>
      </column>
      <item>
       <property name="text">
        <string>foo</string>
       </property>
       <item>
        <property name="text">
         <string>bar</string>
        </property>
        <item>
         <property name="text">
          <string>titi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>toto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>tata</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>bor</string>
        </property>
        <item>
         <property name="text">
          <string>zip</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>zap</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>bur</string>
        </property>
       </item>
      </item>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Tests</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="_checkBox">
           <property name="text">
            <string>test checkbox</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="_comboBox">
           <item>
            <property name="text">
             <string>combo1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>combo2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>combo3</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="_lineEdit"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="_openDialog">
             <property name="text">
              <string>Open Dialog</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="_testLabel">
           <property name="text">
            <string>#</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="_tableWidget">
        <row>
         <property name="text">
          <string>Row1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Row3</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Col1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Col2</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>_treeWidget</zorder>
   <zorder></zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>822</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
